python/src/MultigroupTable.python.cpp

```diff
@@ -34,36 +36,46 @@ void wrapMultigroupTable( python::module& module, python::module& ) {
   table
   .def(
 
-    python::init< std::string, std::string, std::string, std::string,
-                  double, double, double, double,
+    python::init< std::string, std::string, std::string,
+                  double, double, double,
                   EnergyGroupStructure,
                   std::vector< EnergyGroupStructure >,
+                  Velocities,
                   FluxWeights,
                   ReactionCrossSections,
+                  std::optional< std::string >,
+                  std::optional< double >,
+                  std::optional< TotalCrossSection >,
```

Comment:
There should always be a total XS unless it's a secondary particle.

Reply:
That's not what I think the pdf says, but we can change that.

Reply:
The PDF is a little old, so yeah, make it required.

Reply:
I can do that